routes/user.rb

```diff
@@ -68,6 +69,13 @@ class Api
           user.settings['bg_color'] = r.params['bg_color']
           user.settings['font_color'] = r.params['font_color']
           user.settings['red_logo'] = r.params['red_logo']
+          user.settings['white'] = r.params['white']
```

Comment:
let's change all of these...

in model/user.rb add in a const which is like 

SETTINGS = %w[bg_color font_color red_logo white...]

def update_settings(params)
SETTINSGS.each { |setting| settings[setting] = params[setting] }